botocore/data/elb/2012-06-01/waiters-2.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version":2,
+    "waiters":{
+        "InstanceDeregistered": {
```

Comment:
I wonder if we should hold off on this one. I am still not sure if this one is valid in terms of when the ``InvalidInstance`` exception is thrown when multiple instances are provided.